### DISCUSSION TITLE
Designing generic ways to use essence

### DISCUSSION THREAD

**Discussion by pagnabros** [Leafwing-Studios/fop-game#102] [Ideas]
Not a issue per se, but I think it could be cool to have 2-3 ways to use essence that are generic to everything, both player characters and monsters, trying not stepping on the toes of class/feat features. For example, by expending X essence you can augment 2*X damage/absorption.

**Comment by alice-i-cecile** [comment]
Definitely something I'm interested in exploring for feats and gear. I don't really want to add that complexity to the core actions though.
